src/coreclr/nativeaot/Runtime/MathHelpers.cpp

```diff
@@ -106,33 +38,14 @@ EXTERN_C uint64_t QCALLTYPE RhpULMod(uint64_t i, uint64_t j)
     return i % j;
 }
 
-FCIMPL1_D(int64_t, RhpDbl2Lng, double val)
-{
-    return (int64_t)val;
-}
-FCIMPLEND
-
-FCIMPL1_D(int32_t, RhpDbl2Int, double val)
-{
-    return (int32_t)val;
-}
-FCIMPLEND
-
 FCIMPL1_D(uint32_t, RhpDbl2UInt, double val)
```

Comment:
```suggestion
FCIMPL1_D(int32_t, RhpDbl2Int, double val)
{
    return (int32_t)val;
}
FCIMPLEND

FCIMPL1_D(uint32_t, RhpDbl2UInt, double val)
```

Reply:
CoreCLR code needs to be changed too if you want to go back to C++ since it was also switched to the C# variant for this unused helper.

Reply:
Yes, that's fine